### PR TITLE
Add toolbar menu docs, readme and playground

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Add ToolbarMenu examples, api, & playground pages.

### CODE REVIEW

File: src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+
+export const WITHIN_TOOLBAR = `<mat-toolbar color="primary" style="width: 300px">
+    <button mat-icon-button style="margin-right: 16px">
```
Comment (huayunh) [inline src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts:4]:
```suggestion
    <button mat-icon-button style="margin: 0 24px 0 -8px">
```

File: src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div>
+        <div class="mat-title" style="line-height: 1rem">Title</div>
```
Comment (huayunh) [inline src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts:8]: Does not appear centered... 

image

File: src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        <blui-toolbar-menu label="Subtitle">
+            <ng-container blui-toolbar-menu-items>
+                <button mat-menu-item>Menu Item 1</button>
+                <button mat-menu-item>Menu Item 2</button>
+                <button mat-menu-item>Menu Item 3</button>
+            </ng-container>
+        </blui-toolbar-menu>
```
Comment (huayunh) [inline src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts:15]: consider using line highlighting

File: src/docs/pages/component-docs/components/toolbar-menu/toolbar-menu-doc.component.ts [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+import { WITHIN_TOOLBAR } from './examples/within-toolbar.component';
+import { ToolbarMenuPlaygroundKnobs } from './examples/playground.component';
+
+@Component({
+    selector: 'app-toolbar-menu-doc',
+    template: `
+        <app-component-doc-scaffold mdFileName="ToolbarMenu.md" [knobGroups]="knobGroups">
+            <div examples class="app-example">
+                <div class="example-section">
+                    <div class="example-heading">Basic ToolbarMenu</div>
```
Comment (huayunh) [inline src/docs/pages/component-docs/components/toolbar-menu/toolbar-menu-doc.component.ts:15]: Title Case
```suggestion
                    <div class="example-heading">Basic Toolbar Menu</div>
```

**Review by huayunh — COMMENTED** [review]
The other two pages looks good

**Comment by ektasawant** [comment]
Can we change image of toolbar menu ? Right now it is showing border. We can use react docs image.
https://blui-react-docs--pr545-feature-3529-toolbar-n8duejpc.web.app/components/toolbar-menu/api-docs (https://blui-react-docs--pr545-feature-3529-toolbar-n8duejpc.web.app/static/media/ToolbarMenuAnatomy.d6833bfb868b47a80903.png)

Screenshot 2022-09-27 at 11 17 49 AM

**Comment by emclaug2** [comment]
> Can we change image of toolbar menu ? Right now it is showing border. 

Which one?

File: src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts (hunk 3 above)
Comment (emclaug2) [reply src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts:15]: Yep you've got it, added.

File: src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts (hunk 2 above)
Comment (emclaug2) [reply src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts:8]: I've slapped a margin on this one to center the content.

**Comment by ektasawant** [comment]
> > Can we change image of toolbar menu ? Right now it is showing border.
> 
> Which one?

https://github.com/brightlayer-ui/angular-component-library/blob/dev/docs/images/toolbarmenuAnatomy.png This image has left and top borders only. The right and bottom border is missing.

**Comment by emclaug2** [comment]
> https://github.com/brightlayer-ui/angular-component-library/blob/dev/docs/images/toolbarmenuAnatomy.png This image has left and top borders only. The right and bottom border is missing.

Oh I see now, good catch! It's updated now.

**Review by huayunh — APPROVED** [review]

**Pull request merged**